packages/core/test/notification.test.ts

```diff
@@ -0,0 +1,1516 @@
+        title: 'Notification Title',
+        severity: 'WARNING',
+        position: {
+          offset: 0,
+          line: 0,
+          column: 0
+        },
+        category
+      }
+
+      const gqlStatusObject = polyfillGqlStatusObject(rawNotification)
+
+      expect(gqlStatusObject).toEqual(new GqlStatusObject({
+        neo4j_code: rawNotification.code,
+        gql_status: '01N42',
+        status_description: 'warn: warning - unknown warning',
+        diagnostic_record: {
+          OPERATION: '',
+          OPERATION_CODE: '0',
+          CURRENT_SCHEMA: '/',
+          _severity: 'WARNING',
+          _classification: category,
+          _position: {
+            offset: 0,
+            line: 0,
+            column: 0
+          }
+        }
+      }))
+    })
+
+    it.each(getValidCategories())('should polyfill severity INFORMATION', (category) => {
+      const rawNotification = {
+        code: 'Neo.Notification.Warning.Code',
+        title: 'Notification Title',
+        description: 'Description',
+        severity: 'INFORMATION',
+        position: {
+          offset: 0,
+          line: 0,
+          column: 0
+        },
+        category
+      }
+
+      const gqlStatusObject = polyfillGqlStatusObject(rawNotification)
+
+      expect(gqlStatusObject).toEqual(new GqlStatusObject({
+        neo4j_code: rawNotification.code,
+        gql_status: '03N42',
+        status_description: rawNotification.description,
+        diagnostic_record: {
+          OPERATION: '',
+          OPERATION_CODE: '0',
+          CURRENT_SCHEMA: '/',
+          _severity: 'INFORMATION',
+          _classification: category,
+          _position: {
+            offset: 0,
+            line: 0,
+            column: 0
+          }
+        }
+      }))
+    })
+
+    it.each(getValidCategories())('should polyfill severity INFORMATION and no description', (category) => {
+      const rawNotification = {
+        code: 'Neo.Notification.Warning.Code',
+        title: 'Notification Title',
+        severity: 'INFORMATION',
+        position: {
+          offset: 0,
+          line: 0,
+          column: 0
+        },
+        category
+      }
+
+      const gqlStatusObject = polyfillGqlStatusObject(rawNotification)
+
+      expect(gqlStatusObject).toEqual(new GqlStatusObject({
+        neo4j_code: rawNotification.code,
+        gql_status: '03N42',
+        status_description: 'info: informational - unknown notification',
+        diagnostic_record: {
+          OPERATION: '',
+          OPERATION_CODE: '0',
+          CURRENT_SCHEMA: '/',
+          _severity: 'INFORMATION',
+          _classification: category,
+          _position: {
+            offset: 0,
+            line: 0,
+            column: 0
+          }
+        }
+      }))
+    })
+
+    it.each([
+      'UNKNOWN',
+      null,
+      undefined,
+      'I_AM_NOT_OKAY',
+      'information'
+    ])('should polyfill UNKNOWN', (severity) => {
```

Comment:
```suggestion
    ])('should polyfill severity UNKNOWN', (severity) => {
```
+ next test description